content/drafts/access-policies.md

```diff
@@ -0,0 +1,29 @@
+# Access policies
+
+> Access policies is only enabled in some clusters. See which in our [cluster overview](../clusters/README.md).
+
+If you are running your app in a cluster with access policies enabled you can define the access rules for your application.
+If you define none, the policy will **default to deny all incoming and outgoing traffic**.
+
+If you set `allow-all` to `true`, all traffic in the same namespace as the application will be allowed.
```

Comment:
Hvorfor kun samme `namespace`, og ikke samme `cluster`? Er vel sistnevnte som er mest nyttig for folk?

Reply:
Fordi Istio begrenser `allow all` til namespace. Og litt av greia med zero trust er jo å ikke åpne alt.

Reply:
Enig med at `cluster` nok ville ført til at alle hadde det skrudd på.